+ new functionality - get/set user agent

kirillo opened the pull request:
+ new functionality - get/set user agent
+ new functionality - allow webview to identify itself as native browser (BehaveLikeNativeBrowser)

Commit:
+ new functionality - get/set user agent
+ new functionality - allow webview to identify itself as native browser (BehaveLikeNativeBrowser)

neooleg commented on HybridWebControl/Helpers/UserAgentParser.cs:
```diff
@@ -0,0 +1,19 @@
+{
+    public static class UserAgentParser
+    {
+        public static string ConvertToNativeBrowser(this String userAgent)
```
I suggest ConvertUserAgentToNativeBrowser which is long although, but explicit and clear, then safe

neooleg commented on HybridWebControl/Control/HybridWebView.cs:
```diff
@@ -219,6 +232,14 @@ public void SetCookie(string host, string name, string value)
             FutureLoadedPageCookieValue = value;
         }
 
+        public void BehaveLikeNativeBrowser(bool shouldBeNative)
```
We don't need this, there is extension method

Commit:
Code refactoring due to comments in pull request #16
+ remove BehaveLikeNativeBrowser
+ webview behaviour can be changed using UserAgent.ConvertUserAgentToNativeBrowser method
